roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -325,6 +325,24 @@ def get_major_minor_version(self, openshift_image_tag=None):
 
         return tuple(int(x) for x in components[:2])
 
+    def get_required_version(self, name, version_map):
+        """Return the correct required version(s) for the current (or nearest) OpenShift version."""
+        openshift_version = self.get_major_minor_version()
+
+        earliest = min(version_map)
+        latest = max(version_map)
+        if openshift_version < earliest:
+            return version_map[earliest]
+        if openshift_version > latest:
```

Comment:
You could change `<` and `>` to `<=` and `>=`, respectively, but this is fine as is.